driver/xrt/include/accl.hpp

```diff
@@ -727,6 +751,22 @@ class ACCL {
    */
   std::string dump_communicator();
 
+  /**
+   * Return CCLO address of communicator.
+   *
+   * @param ACCL::communicatorId Numerical ID of the target communicator.
+   * @return addr_t Address of the communicator in CCLO memory.
+   */
+  addr_t get_communicator_adr(communicatorId comm_id=GLOBAL_COMM);
```

Comment:
Maybe it makes sense to name this `get_communicator_addr` to match with the arithmetic config equivalent?